src/ILCompiler.Compiler/src/project.json

```diff
@@ -3,7 +3,9 @@
     "NETStandard.Library": "1.6.0",
     "System.IO.MemoryMappedFiles": "4.0.0",
     "System.Reflection.Metadata": "1.4.1-beta-24227-04",
-    "System.Runtime.CompilerServices.Unsafe": "4.0.0"
+    "System.Runtime.CompilerServices.Unsafe": "4.0.0",
+    "System.Security.Cryptography.Algorithms": "4.2.1-beta-24223-07",
```

Comment:
Do you really need beta version of the package? There is stable version that you can use (4.2.0 from https://www.nuget.org/packages/System.Security.Cryptography.Algorithms/).